docs/manual/introduction.tex

```diff
@@ -792,6 +792,10 @@
 Print information about the git repository from which the Checker Framework
 was compiled.
 
+\item
+ \<-AdumpOnErrors>
+Prints the stack trace when the checker framework encounters an error.
```

Comment:
`checker framework` should be `Checker Framework` to be consistent.
`Also output a stack trace when errors or warnings are reported.`

Reply:
I have made the changes in my latest commit